samples/sampler/stories/default/Checkbox.js

```diff
@@ -0,0 +1,32 @@
+import {action} from '@enact/storybook-utils/addons/actions';
```

Comment:
@ybsung I remember you've removed Checkbox story.. Are we OK to add this?

Reply:
Should I wait until YB comes back with an answer for this issue?

Reply:
Let me check with others..

Reply:
We are good to add this story again. Thanks!